engine/servers/pgvector-single-node/docker-compose.yaml

```diff
@@ -3,13 +3,17 @@ version: '3.7'
 services:
   pgvector:
     container_name: pgvector
-    image: ankane/pgvector:v0.5.1
+    image: pgvector/pgvector:0.6.0-pg16
```

Comment:
Can you please use the official container?

Reply:
This is the official image starting with 0.6.0. https://github.com/pgvector/pgvector#docker-1